project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.18
+sbt.version=1.3.6
```

Comment:
Ah, I didn't notice this in previous PR. The latest sbt version is `1.3.13`

Reply:
Yes, it is. But there are some silly bugs that occur on project import. Spent some time on research to figure out `1.3.6` is the last version without this issue.